saleor/tests/e2e/utils.py

```diff
@@ -18,3 +20,22 @@ def assign_permissions(api_client, permissions):
         app = api_client.app
         if app:
             app.permissions.add(*permissions)
+
+
+def request_matcher(r1, r2):
+    body1 = json.loads(r1.body.decode("utf-8"))
+    body2 = json.loads(r2.body.decode("utf-8"))
+
+    # Check if all key-value pairs in body1 are present in body2
```

Comment:
```
if set(body1.keys()) != set(body2.keys()):
    return False
```
Then you are sure, that body1 and body2 have the same keys. So only need to check b1.value vs b2.value